docs/src/tutorial.md

```diff
@@ -208,4 +208,23 @@ end
 
 Because such usage necessarily requires passing concrete types to your functions, calling `@report_call exercise_mypkg()` leads to more precise analysis than `report_package`.
 
-Furthermore, once you have written a function like `exercise_mypkg`, you can use a package like [`SnoopPrecompile`](https://github.com/timholy/SnoopCompile.jl) to precompile the function, which will thus precompile all code exercised in the function, significantly reducing your package's latency.
+Furthermore, once you have written a function like `exercise_mypkg`, you can use a package like [`PrecompileTools`](https://github.com/JuliaLang/PrecompileTools.jl) to precompile the function, which will thus precompile all code exercised in the function, significantly reducing your package's latency.
+
+Conversely, if you've already implemented a precompile workload, you can do the following:
+
+```julia
+using MyPkg, JET, MethodAnalysis
+mis = methodinstances(MyPkg)    # get all the compiled methodinstances for functions owned by the package
+# Now let's filter out the ones that pass without issue
+hasreport(report) = report !== nothing && !isempty(JET.get_reports(report))
+badmis = filter(mis) do mi
+    hasreport(report_call(mi.specTypes))
+end
+```
+
+Then you can inspect the methodinstances in `badmis` individually with `report_call(mi.specTypes)`.
```

Comment:
Maybe we want to make `report_call` be able to take `mi::MethodInstance` directly?
We can do something like:
```diff
diff --git a/src/JET.jl b/src/JET.jl
index 3d5bcfc7..87103515 100644
--- a/src/JET.jl
+++ b/src/JET.jl
@@ -762,6 +762,15 @@ function analyze_and_report_call!(analyzer::AbstractAnalyzer, @nospecialize(tt::
     source = lazy"$analyzername: $sig"
     return JETCallResult(result, analyzer, source; jetconfigs...)
 end
+function analyze_and_report_call!(analyzer::AbstractAnalyzer, mi::MethodInstance,
+                                  jetconfigs...)
+    validate_configs(analyzer, jetconfigs)
+    analyzer, result = analyze_method_instance!(analyzer, mi)
+    analyzername = nameof(typeof(analyzer))
+    sig = LazyPrinter(io::IO->Base.show_tuple_as_call(io, Symbol(""), mi.specTypes))
+    source = lazy"$analyzername: $sig"
+    return JETCallResult(result, analyzer, source; jetconfigs...)
+end
 
 # TODO `analyze_builtin!` ?
 function analyze_gf_by_type!(analyzer::AbstractAnalyzer, @nospecialize(tt::Type{<:Tuple}))
```
and then `report_call` can analyze `mi::MethodInstance`.